airgun/widgets.py

```diff
@@ -627,7 +627,7 @@ def fill(self, item):
 
 class Search(Widget):
     """Searchbar for table filtering"""
-
+    ROOT = '//div[@id="search-bar" or contains(@class, "toolbar-pf-filter")]'
     search_field = TextInput(
```

Comment:
@tpapaioa thanks for fixing this, the search seems to be working for me.
But I am facing one weird issue, when I am on read page of a repository, and try to create new repo then it navigates to settings page instead, and fails with `*** selenium.common.exceptions.NoSuchElementException: Message: Could not find an element Locator(by='xpath', locator=".//span[contains(@class, 'editable-click')]")
` error. Not sure if I did anything wrong.

Reply:
tried again, something wrong with my test only, normally this is working fine. Need to debug. :|

Reply:
That error appears to be an issue with finding the following element in `airgun/widgets.py`:

```
class PopOverWidget(Widget):
[...]
    ROOT = '.'
    column_value = Text(".//span[contains(@class, 'editable-click')]")
```

That widget is used in `airgun/views/settings.py`:

```
class SettingsView(BaseLoggedInView, SearchableViewMixin):
    title = Text("//h1[text()='Settings']")
    table = Table(
        './/table',
        column_widgets={'Value': PopOverWidget()},
    )
```

I don't think that the change to the `Search` widget would affect this usage of `Table` / `PopOverWidget`, but if you see the error again I'll be happy to look into it.

Reply:
yes, it's not because of your changes. Created https://github.com/SatelliteQE/airgun/issues/546, figured it out what's the root cause and how to fix it.